src/mock_mev/mock_mev_launcher.star

```diff
@@ -14,8 +14,9 @@ def launch_mock_mev(plan, el_uri, beacon_uri, jwt_secret):
 			cmd = [
 				"--jwt-secret={0}".format(jwt_secret),
 				"--el={0}".format(el_uri),
-				"--cl={0}".format(beacon_uri)
+				"--cl={0}".format(beacon_uri),
+				"--log-level=debug"
```

Comment:
Do you want to hardcode `debug` or use the value of `global_client_log_level`?

Reply:
Oh that would be much better! Let me change it :)